docs/transpile/custom_passes.mdx

```diff
@@ -0,0 +1,101 @@
+
+```python
+    qc = EfficientSU2(12, entanglement='circular', reps=1)
+    qc_t = transpile(qc, target=target, seed_transpiler=12345)
+    qc_t.draw('mpl', style='iqp', fold=-1, idle_wires=False)
+```
+![Ansatz transpiled to `FakeSherbrooke`](/images/transpile/custom_passes/qc_transpiled.png)
+
+A _dynamical decoupling sequence_ is a series of gates that compose to the identity and are spaced regularly in time. They flip the qubits around the Bloch sphere which changes the effect of noise channels, therefore suppressing decoherence. For a full description, see [A Quantum Engineer's Guide to Superconducting Qubits](https://arxiv.org/abs/1904.06560). These are similar to refocusing pulses used in nuclear magnetic resonance. Here we'll create a simple sequence called XY4 consisting of four gates.
+
+```python
+    X = XGate()
+    Y = YGate()
+
+    dd_sequence = list((X, Y, X, Y))
+```
+
+Because of the regular timing of dynamical decoupling sequences, we must add information about the `YGate` to the `target` because it is *not* a basis gate, whereas the `XGate` is. We know *a priori* that the `YGate` has the same duration and error as the `XGate`, however, so we can just retrieve those properties from the `target` and add them back for the `YGates`s. This is also why we needed to save the `basis_gates` separately, since we are adding the `YGate` instruction to the `target` although it is not an actual basis gate of `FakeSherbrooke`.
+
+```python
+    y_gate_properties = {}
+    for qubit in range(target.num_qubits):        
+        y_gate_properties.update({
+            (qubit,): InstructionProperties(
+                duration = target['x'][(qubit,)].duration,
+                error = target['x'][(qubit,)].error)})
+
+    target.add_instruction(YGate(), y_gate_properties)
+```
+
+Ansatz circuits such as `EfficientSU2` are parameterized, so they much have value bound to them before being sent to the backend. Here we use `.assign_parameters` to attach random values generated by `numpy.random`.
+
+```python
+    qc_t.assign_parameters(np.random.uniform(-np.pi, np.pi, qc_t.num_parameters), inplace=True)
+```
+
+Next, we execute the custom passes. We instantiate the `PassManager` with `ALAPScheduleAnalysis` and `PadDynamicalDecoupling`.  We must run `ALAPScheduleAnalysis` first to add timing information about the quantum circuit before the regularly-spaced dynamical decoupliong sequences can be added. These passes are run on the circuit with `.run()`.
+
+```python
+    pm = PassManager([
+        ALAPScheduleAnalysis(target=target),
+        PadDynamicalDecoupling(target=target, dd_sequence=dd_sequence),
+        ])
+    qc_dd = pm.run(qc_t)
+```
```

Comment:
I feel like this code block is the most important part of this page, I think it would be good to emphasise it somehow and make the point to the reader that this is the basic structure for constructing and running a custom passmanager, and where they could go to look at the other qiskit passes that are available. 

Possibly the page introduction should start with a code block like this (maybe just pseudo code) as a way to explain in general how to construct/run the PassManager object, and highlight the different passes available. Then launch into the worked example for dynamical decoupling.

Reply:
I rewrote the whole page.